api/modules/zomato/zomato_response.py

```diff
@@ -0,0 +1,25 @@
+class Zomato_Response(object):
```

Comment:
Please follow [PEP8 convention](https://www.python.org/dev/peps/pep-0008/?#class-names) for naming 
Class names should follow `CapWords` convention